### PR TITLE
Adjust Several Items for Post-Beta

### PR DESCRIPTION
**Pull request by wgranger**
This PR makes ~three~ four minor tweaks to master.

- Make sure the `WorkflowPrompt` component successfully mounts and unmounts on route change
- Tighten up button margins on main page workflow buttons
- Remove pointer cursor from inactive workflow buttons on landing page
- Updated package-lock.json

That should be about it.

### CODE REVIEW

**Comment by shaunanoordin** [comment]
That's _four_

**Comment by wgranger** [comment]
Oh, haha. I added the fourth on a whim.

File: src/ducks/work-in-progress.js [hunk 1]
```diff
@@ -154,6 +154,8 @@ const check = (user) => {
   const workflowId = localStorage.getItem(`${userId}.${WORKFLOW_ID_KEY}`);
   const subjectId = localStorage.getItem(`${userId}.${SUBJECT_ID_KEY}`);
   const annotations = localStorage.getItem(`${userId}.${ANNOTATIONS_KEY}`);
+  console.log(!!workflowId && !!subjectId && !!annotations);
```
Comment (shaunanoordin) [inline src/ducks/work-in-progress.js]: I assume these `console.log()`s aren't necessary any more.

**Comment by shaunanoordin** [comment]
BAM! Done. Now let's meeerge

**Pull request merged**